src/slack/actions/supportRequest.ts

```diff
@@ -37,7 +37,7 @@ export const supportRequest: Middleware<SlackActionMiddlewareArgs<BlockAction>>
       const requestItem = new SupportRequest(
         slackId,
         slackName,
-        actionId === actionIds.joinIdeaPitchRequestQueue ? SupportRequestType.IdeaPitch : SupportRequestType.TechnicalSupport,
+        actionId === actionIds.joinIdeaPitchRequestQueue ? SupportRequestType.IdeaPitch : (actionId === actionIds.joinTechnicalRequestQueue ? SupportRequestType.TechnicalSupport : SupportRequestType.JobChat)
```

Comment:
For both of these ternary operators you'll want to make a variable above and have normal if statements to decide what to set it to. Since linting is upset that there are nested ternaries. (Same with the other one)